src/components/common/Layout/components/BackButton.tsx

```diff
@@ -7,15 +7,16 @@ import { PropsWithChildren } from "react"
 
 type Props = {
   href?: string
+  forceRender?: boolean
```

Comment:
Instead of this new prop, we should separate the component into a general BackButton and a BackToExplorer button, and use the latter where we used BackButton so far, and use the simple BackButton for the new dashboard navigation